### PR TITLE
Add nose-progressive

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

File: recipes/nose-progressive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - nose>=1.2.1
```
Comment (synapticarbors) [inline recipes/nose-progressive/meta.yaml]: You need a space between `nose` and `>=`

**Comment by synapticarbors** [comment]
The license file isn't included in the source distro since it isn't listed in the `MANIFEST.in`. You can manually package it:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and put in an issue, although that project hasn't been updated in a long time so I'm not sure if it will get fixed.

**Comment by sannykr** [comment]
@dougalsutherland : I am trying to skip windows build because it requires "blessings" that is not compatible with windows, but it doesn't skip windows build. Could you please look at the skip statement and let me know if there is anything wrong with that.

**Comment by dougalsutherland** [comment]
@sannykr It's correct, don't worry about it.

Explanation, if you care:
- On most of the Windows builds, it just says "hey, nothing to do, I'm good": [example](https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.17037/job/kanarv8fr22pqsit#L160). This is normal on staged-recipes (wouldn't happen in a feedstock).
- Windows builds on python 3.5 were temporarily broken in the build system, before it even got to your recipe.
  - The problem was: it had an old version of the `conda` package that doesn't support `noarch: python`.
  - `conda` depends on `six`.
  - The `six` package was made `noarch: python` (https://github.com/conda-forge/six-feedstock/pull/8).
  - So when it upgraded `conda`, it messed up its installation of `six`, which broke the build system.
  - This was undone in https://github.com/conda-forge/six-feedstock/pull/10, so it's fixed now.

I hit restart on the AppVeyor build so that the red X will go away. :)

**Comment by sannykr** [comment]
@dougalsutherland : Thanks.

**Comment by sannykr** [comment]
@jakirkham : Could you please review and merge if everything looks okay.

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : Could you please review and merge if everything looks okay.

**Pull request merged**